### DISCUSSION TITLE
Wrap text in compact mode

### DISCUSSION THREAD

**Discussion by javiavid** [RafaelGB/obsidian-db-folder#310] [Announcements]
It would be possible that the option of multiple lines independent of the mode (normal, compact, wide)?

This will allow you to have a compact mode and see multiple lines.

**Comment by RafaelGB** [comment]
This could be difficult, I will move it to a discussion

**Comment by RafaelGB** [comment]
Now I think we have to focus into another details right now and this issue is a potencial breaker of the actual compact mode

**Comment by javiavid** [comment]
I understand perfectly, maybe in the future it will be possible.
